artifacts/artifacts.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/NethermindEth/starknet.go/types"
 )
 
+//go:embed hello_starknet_compiled.casm.json
+var ExampleWorldCasm []byte
+
+//go:embed hello_starknet_compiled.sierra.json
+var ExampleWorldSierra []byte
```

Comment:
Seems like you use these variables only in tests. Most likely they will end up in a final binary even though they are not used there. You can verify it with `strings <binary> | grep <content from json>`

Reply:
I've moved them to new tests folders so they don't get included in the final binary